sdk/openai/openai/package.json

```diff
@@ -37,10 +37,14 @@
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
     "build:samples": "echo Obsolete",
     "build:output": "node scripts/renameOutput.mjs",
-    "build": "tsc -p . && dev-tool run bundle && api-extractor run --local && npm run build:output",
-    "build:test": "npm run build",
+    "generate": "pwsh ../../../eng/common/scripts/TypeSpec-Project-Sync.ps1 . && npm run copy-typespec-config && cd TempTypeSpecFiles/OpenAI.Inference && tsp install && tsp compile main.tsp && cd ../.. && rimraf sources/generated && npm run copy-generated",
```

Comment:
if this does more than just call sync / generate then we should have a sync and discuss the gaps.

Engsys is currently relying on the fact that they can simply call sync then generate and produce the same results.

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/26138/commits/c5372c92dc116f8296cff1a1423d2a85034426cb